libs/datasets/timeseries.py

```diff
@@ -310,6 +311,18 @@ def timeseries_wide_dates(self) -> pd.DataFrame:
         )
         return timeseries_wide
 
+    @lru_cache(maxsize=None)
+    def timeseries_wide_dates_variable_first(self) -> pd.DataFrame:
+        return self.timeseries_wide_dates().reorder_levels(
+            [PdFields.VARIABLE, CommonFields.LOCATION_ID]
+        )
+
+    @lru_cache(maxsize=None)
+    def timeseries_wide_dates_variable_first_diff(self, *, periods: int) -> pd.DataFrame:
```

Comment:
this feels pretty specific to the test positivity dataset calculation - thoughts on just adding this where it's called?